unime/src/routes/credentials/[id]/DataUrlImageRenderer.svelte

```diff
@@ -0,0 +1,13 @@
+<script lang="ts">
+  export let key: string;
+  export let dataUrl: string;
+</script>
+
+<!--
+@component
+This renderer for `credentialSubject` fields renders an embedded image for a given [Data URL](https://developer.mozilla.org/en-US/docs/Web/URI/Schemes/data).
+-->
+<div class="flex flex-col items-center rounded-xl bg-background-alt px-4 py-3 text-[13px]/[24px]">
+  <h2 class="w-full font-medium text-text-alt">{key}</h2>
+  <img src={dataUrl} alt="embedded" class="h-auto w-32 rounded-lg" />
```

Comment:
Image should be left aligned because everything else is left aligned. Also are we sure images will always have a 1:1 ratio? Constraining the height makes sense but it should probably be accompanied by `object-contain` like in the credential logo.

Reply:
I thought it would look better if the image is centered, since it visually aligns better with the credential image in the header. I now left aligned it, until we have a clearer idea of how and where we want to display embedded images.